assignment/scripts/function-practice.js

```diff
@@ -87,3 +111,34 @@ function sumAll( ) {
 // 11. Pick a problem from Edabit(https://edabit.com/) or 
 //     CodeWars(https://www.codewars.com/). Then describe it 
 //     here in a comment, write the function, and test it!
+
+//From Edabit: Find the Index Part 1
+//Create a function that finds the index of a given item.
+
+//Examples
+//search([1, 5, 3], 5) ➞ 1
+
+//search([9, 8, 3], 3) ➞ 2
+
+//search([1, 2, 3], 4) ➞ -1
+//Notes
+//If the item is not present, return -1.
+
+/* My Comments:
+
+In this problem you are given a function called search that is called in which you are looking
+for a particular number within the given array. If the number is found, it will
+give you a positive number. If false, then a -1.
+*/
+// Unfinished
+function search(array, item) {
+  if (item === array.indexOf(index)) {
+    return [index];
```

Comment:
Whoops! This will throw a reference error. Make sure you are testing your code and not turning in assignments which cannot run.